crates/containerd-shim-wasmedge/Cargo.toml

```diff
@@ -18,7 +18,7 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 nix = { workspace = true }
 libc = { workspace = true }
-libcontainer = { workspace = true }
+libcontainer = { workspace = true, features = ["default"] }
```

Comment:
**question:** Shouldn't this go through `containerd-shim-wasm = { workspace = true, features = ["libcontainer", ...] }` to make sure that `containerd-shim-wasm` has `libcontainer` enabled ?

Reply:
containerd-shim-wasm, by default, depends on libcontainer. I am not entirely if I understand your question.

Reply:
my concern was that something like this
```toml
[package]
name = "my_crate"
version = "0.1.0"

[dependencies]
containerd-shim-wasm = { path = "../runwasi/crates/containerd-shim-wasm", default-features = false}
containerd-shim-wasmedge = { path = "../runwasi/crates/containerd-shim-wasmedge", default-features = false}
```

would not enable `libcontainer` in `containerd-shim-wasm`, but it does :-)

Reply:
I think my last fix has resolved this.